internal/resources/resource_host.go

```diff
@@ -460,12 +485,17 @@ func resourceMetalHostRead(d *schema.ResourceData, meta interface{}) (err error)
 			vTargetIQN:   i.TargetIQN,
 		}
 		volumeInfos = append(volumeInfos, vi)
+		discoveryIPs = append(discoveryIPs, i.DiscoveryIP)
 	}
 
 	if err := d.Set(hVolumeInfos, volumeInfos); err != nil {
 		return err
 	}
 
+	if err := d.Set(hDiscoveryIPs, removeDuplicateStr(discoveryIPs)); err != nil {
```

Comment:
How are you getting a  list of discoveryIps from Quake? A list of discovery Ips?
Should we  say which volume should use which discovery IP? I was thinking more of lines [key, value] for [Volume ID, disocvery IP]

Reply:
Was able to show VMaaS and Linux to use just 1 DiscoveryIP